py/selenium/webdriver/firefox/service.py

```diff
@@ -28,7 +28,7 @@ class Service(service.Service):
 
     def __init__(self, executable_path: str = DEFAULT_EXECUTABLE_PATH,
                  port: int = 0, service_args: List[str] = None,
-                 log_path: str = "geckodriver.log", env: dict = None):
+                 log_path: Union[str, Tuple[str, str]] = "geckodriver.log", env: dict = None):
```

Comment:
`Sequence[str, str]` is probably preferred here; a list would be fine etc, let's not rely on concrete implementations when we can avoid it (I know this statement is grossly untrue for other parts but i'd like to keep anything new as loose as we can get away with, checking the `isinstance` will need an update too

Reply:
Done!